run.js

```diff
@@ -11,68 +12,120 @@ const pathFor = mode => path.join(
   'react-dom.' + mode + '.js'
 );
 
+// set up AST files.
 const ReactDOMProd = pathFor('production.min');
 const ReactDOMDev = pathFor('development');
 
-console.time('parse prod');
 const prod = fs.readFileSync(ReactDOMProd, 'utf8');
 const prodAst = babylon.parse(prod);
-console.timeEnd('parse prod');
+fs.writeFileSync('react-dom.production.min.ast', 'module.exports = ' + JSON.stringify(prodAst) + ';');
+fs.writeFileSync('react-dom.production.min.js', 'module.exports = ' + JSON.stringify(prodAst) + ';');
+fs.writeFileSync('react-dom.production.min.json', JSON.stringify(prodAst));
```

Comment:
It's not inside a test function, so I figured it didn't really matter in terms of results. If it matters to you for overall run time, happy to change it.